website/modules/@apostrophecms/form/lib/GoogleSheetsErrorHandler.js

```
@@ -0,0 +1,37 @@
+
+class GoogleSheetsErrorHandler {
+  constructor(logger) {
+    this.logger = logger;
+  }
+
+  static getErrorMessage(errorCode) {
+    return ERROR_MESSAGES[errorCode] || null;
+  }
+
+  static formatError(error, context, spreadsheetId = null) {
+    const predefinedMessage = this.getErrorMessage(error.code);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)